packages/redux-subspace/src/actions/processAction.js

```diff
@@ -8,9 +8,10 @@
 
 import hasNamespace from '../actions/hasNamespace'
 import isGlobal from '../actions/isGlobal'
+import isReduxSpecific from '../actions/reduxSpecific'
 
 const processAction = (namespace) => (action, callback, defaultValue) => {
-    if (!namespace || isGlobal(action)) {
+    if (!namespace || isGlobal(action) || isReduxSpecific(action)) {
```

Comment:
I'd be tempted to move this into the `isGlobal` check.  Essentially we are saying that redux actions are _always_ global actions.